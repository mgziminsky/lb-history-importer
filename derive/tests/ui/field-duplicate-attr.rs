use lb_importer_derive::IntoPayload;

#[derive(IntoPayload)]
struct DuplicateError {
    #[track]
    #[track]
    track: String,
    other: String,
}

fn main() {}
