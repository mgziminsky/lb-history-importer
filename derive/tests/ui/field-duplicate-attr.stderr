error: Duplicate attribute
 --> tests/ui/field-duplicate-attr.rs:6:5
  |
6 |     #[track]
  |     ^^^^^^^^

error: Duplicate attribute
 --> tests/ui/field-duplicate-attr.rs:5:5
  |
5 |     #[track]
  |     ^^^^^^^^
